tensorflow/tools/ci_build/install/install_pip_packages.sh

```diff
@@ -95,15 +95,15 @@ pip3 install --upgrade gast
 pip3 install --upgrade termcolor
 
 # Keras
-pip3 install keras-nightly --no-deps
+pip3 install keras-nightly==2.6.0.dev --no-deps
 pip3 install keras_preprocessing==1.1.0 --no-deps
 pip3 install --upgrade h5py==3.1.0
 
 # Estimator
-pip3 install tf-estimator-nightly --no-deps
+pip3 install tf-estimator-nightly==2.6.0.dev --no-deps
 
 # Tensorboard
-pip3 install tb-nightly --no-deps
+pip3 install tb-nightly==2.6.0.a --no-deps
```

Comment:
This will not work as it will not find any exact version. Use the same string as in the mentioned forum line.